TerrariaServerAPI/TerrariaApi.Server/EventArgs/SetDefaultsEventArgs.cs

```diff
@@ -15,5 +16,7 @@ public F Info
 			get;
 			set;
 		}
+
+		public ItemVariant ItemVariant { get; set; }
```

Comment:
I'm just guessing this needs to be documented in the tshock changelog too?

Reply:
Yep, i'm not yet certain how this is used, could be an override, since ItemVariant can be also changed within the vanilla code after this runs, but this was added to reflect the vanilla signatures - who knows, plugins can alter this now too (especially with On.* hooks!)
Added in: https://github.com/Pryaxis/TShock/pull/2670/commits/24c0f106b85ae5e8f039e82047975d3f8c8ad3f1